### PR TITLE
add nowsms as a provider

### PR DESCRIPTION
**Pull request by khaledkbadr**
related to #67

### CODE REVIEW

**Comment by marcelcorso** [comment]
thanks @khaledkbadr and @mootezbessifi  for the PR! 
I have a few comments....

File: examples/config.yaml [hunk 1]
```diff
@@ -42,6 +42,12 @@ providers:
     password: 'password'
   pushbullet:
     access_token: 'o.AbCdEfGhIjKlMnOpQrStUvWxYz012345'
+  nowsms:
+    username: 'username'
+    password: 'password'
+    phone_numbers:
```
Comment (marcelcorso) [inline examples/config.yaml]: You should specify the phone numbers that will receive the SMS on the `To` attribute of the receivers. Like this https://github.com/messagebird/sachet/blob/master/examples/config.yaml#L52. 

And not have another configuration option. This is more consistent and eases the move from one privider to another

File: provider/nowsms/nowsms.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	request, err := http.NewRequest("POST", nowSmsURL, nil)
+	if err != nil {
+		return err
+	}
+	request.Header.Set("User-Agent", "Sachet")
+
+	params := request.URL.Query()
+	params.Add("User", ns.User)
+	params.Add("Password", ns.Password)
+	params.Add("PhoneNumber", strings.Join(ns.PhoneNumbers, ","))
```
Comment (marcelcorso) [inline provider/nowsms/nowsms.go]: this data should come from the `message.To` not `ns`

**Comment by khaledkbadr** [comment]
@marcelcorso thanks for feedback, I've resolved your comments. Please recheck and if anything else found please add more comments :)

**Comment by mootezbessifi** [comment]
thanks @khaledkbadr  for such contribution.

**Pull request merged**